### ISSUE TITLE
Since flag is broken

### ISSUE THREAD

**Issue by sn3d** [unravela/gitwrk#1]
When I type:

`gitwrk --since 2020-07-04`

I see also records older than 04.07.2020:

```
| 2020-07-08 02:52:25 | zdenko.vrabel@unravela.io | chore |       |  7h30m0s |
| 2020-07-04 18:19:08 | zdenko.vrabel@unravela.io | chore |       |     5m0s |
| 2020-07-03 10:44:18 | zdenko.vrabel@unravela.io | chore |       |   6h0m0s |
| 2020-07-02 10:44:18 | zdenko.vrabel@unravela.io | chore |       |   6h0m0s |
```

**Comment by sn3d** [comment]
This is caused because commit in 2020-07-04 have multi-spent:

`spent 5m 6h 6h`

That means even commits are from 2020-07-04, but logged workhours are before 2020-07-04

**Comment by sn3d** [comment]
Fixed in 1.0.9
